### PR TITLE
chore(front): Various visual fixes

### PR DESCRIPTION
**Pull request by cpvannier**
## Description

- Various visual fixes, including:
  - Colors
  - Fonts
  - Padding values
  - z-index values
- Return to a dataset's main page by clicking on its name in the header

### CODE REVIEW

File: ui/components/imageWorkspace/src/components/Features/FeatureInputs.svelte [hunk 1]
```diff
@@ -29,7 +29,7 @@
 {#each features as feature}
   <div class="mt-1">
     {#if isEditing || feature.value !== undefined}
-      <p class="font-medium pb-1">{feature.label}</p>
+      <p class="font-bold first-letter:uppercase pb-1">{feature.label.replace("_", " ")}</p>
```
Comment (timothee-LJN) [inline ui/components/imageWorkspace/src/components/Features/FeatureInputs.svelte:32]: that's nice !

File: ui/components/imageWorkspace/src/components/ActionsTabs/SceneTabContent.svelte [hunk 2]
```diff
@@ -79,19 +79,19 @@
...
         <p>{meta.width}</p>
       </div>
       <div class="grid gap-4 grid-cols-[100px_auto] mt-2">
-        <p class="font-light first-letter:uppercase">Height</p>
+        <p class="font-bold first-letter:uppercase">Height</p>
         <p>{meta.height}</p>
       </div>
       <div class="grid gap-4 grid-cols-[100px_auto] mt-2">
-        <p class="font-light first-letter:uppercase">Format</p>
+        <p class="font-bold first-letter:uppercase">Format</p>
```
Comment (timothee-LJN) [inline ui/components/imageWorkspace/src/components/ActionsTabs/SceneTabContent.svelte:94]: in my opinion this is slightly too bold compared with the rest of the page

**Review by timothee-LJN — APPROVED** [review]
nice finetunning !

**Review by BertrandRenault — APPROVED** [review]

File: ui/components/imageWorkspace/src/components/ActionsTabs/SceneTabContent.svelte (hunk 2 above)
Comment (cpvannier) [reply ui/components/imageWorkspace/src/components/ActionsTabs/SceneTabContent.svelte:94]: It looks about the same as the template on Sketch, but we will adjust later if necessary

**Pull request merged**